tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -5783,7 +5783,7 @@ Status ParseEquation(OpConverterParams* params,
   OperandLabelCounts input_label_counts;
   LabelCounts output_label_counts;
   absl::InlinedVector<bool, 2> input_has_ellipsis;
-  bool output_has_ellipsis;
+  bool output_has_ellipsis{false};
```

Comment:
Good catch! It seems to me that there are similar problem with input_has_ellipsis. Checking with TensorFlow people on this.